python/examples/tools/python_tool.py

```diff
@@ -0,0 +1,31 @@
+import asyncio
+import sys
+import traceback
+
+from beeai_framework.adapters.ollama.backend.chat import OllamaChatModel
+from beeai_framework.agents.react.agent import ReActAgent
+from beeai_framework.errors import FrameworkError
+from beeai_framework.memory import UnconstrainedMemory
+from beeai_framework.tools.python import PythonTool
+from beeai_framework.tools.storage import Input, LocalPythonStorage
+
+
+async def main() -> None:
+    llm = OllamaChatModel("llama3.1")
+    p = PythonTool(
+        {
+            "codeInterpreter": {"url": "http://127.0.0.1:50081"},
```

Comment:
Since this requires an external service to be running it will cause the e2e tests to fail, you should add this example to the excludes list in `tests/examples/test_examples.py` with a reason comment